### PR TITLE
Add scripts and doc to generate pdf

### PR DESCRIPTION
**Pull request by alemorvan**
#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

**Comment by alemorvan** [comment]
warning: I added a logo needed for the cover of the book

The idea is to generate all books at each month release.
What do you think about that @NeilHanlon? Do you think it's possible?

I also would communicate about the fact one can generate the pdf he wants from our source and make its own personnalized document.

File: build_pdf/README.MD [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# How to generate your own book
+
+## Installation
+
+Fist, fork this repo.
```
Comment (SergeCroise) [inline build_pdf/README.MD:5]:
```suggestion
First, fork this repo.
```

File: build_pdf/README.MD [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+```
+
+Then run the build process:
+
+```bash
+cd build_pdf
+mkdocs build -f path_to_config_file.yml
+```
+
+or launch the script bash to build everything:
```
Comment (SergeCroise) [inline build_pdf/README.MD:20]:
```suggestion
or launch the bash script to build everything:
```

**Comment by lumarel** [comment]
@alemorvan  This seems to be a good case to just convert the whole thing to a Github Action, which dumps it's output in an buildartifact
plus for simplification using mkdocs's configuration inheritance feature to not duplicate half of the configuration file 🙂 

And everyone who forks the repo can enable this workflow in their fork as well if they need it.
(if that sounds good I can generate you a draft how that would look)

**Comment by alemorvan** [comment]
That would be amazing indeed! Thx @lumarel 
I could do that on gitlab-ci, but not on github action.
I want to see what it would look like.
And indeed, whoever wants to make his own PDF would just have to fork the repository.
My original idea was to launch the CI only on new releases.

**Comment by alemorvan** [comment]
@lumarel Thx for your idea.
I read the documentation about inheritance  and applied it to my PR.
I also split the config files thanks to the book which is concerned and added a loop in the bash script to make it more dynamic.

I miss a solution to add a month or a version in the front page.
I will have to think about it.

**Comment by lumarel** [comment]
@alemorvan There goes the time, sorry for the late reply 🙂 

Okay great! Worked even better than I thought it would 😄 
ah oh you want to publish it on the front page, hm don't know if there is an easy way, but as I said, Github Actions, and available in the artifacts there is easily possible, even with a schedule!

Will send you a PR on your branch soon 🙂

**Review by sspencerwire — APPROVED** [review]

**Pull request merged**